src/site/xdoc/release_notes.xml

```diff
@@ -21,6 +21,7 @@
        <ul>
           <li>WatchedCDXSource now monitors directories for new CDX files. <a href="https://github.com/iipc/openwayback/pull/181">#181</a></li>
           <li>Minor fixes to replace hardcoded port numbers aun URL prefixes with placeholders. <a href="https://github.com/iipc/openwayback/pull/223">#223</a></li>
+          <li>MAVEN compile and build now requires Java 7.<a href="https://github.com/iipc/openwayback/pull/181">#178</a></li>
```

Comment:
This links to the wrong issue. And the text should really just be "Require Java 7" This also means that Java 7 is needed to run OWB.